site/src/lib/blocks.ts

```diff
@@ -161,11 +161,15 @@ export const excludeHiddenBlocks = (
   blocks: ExpandedBlockMetadata[],
 ): ExpandedBlockMetadata[] => {
   return blocks.filter(
-    ({ packagePath, protocol }) =>
-      !blocksToHide.includes(packagePath) && parseFloat(protocol) >= 0.2,
+    ({ packagePath }) => !blocksToHide.includes(packagePath),
   );
 };
 
+/** Blocks that aren't compliant with BP V0.2  */
+export const isNonV0_2Block = ({ protocol }: ExpandedBlockMetadata) => {
```

Comment:
we could call this `checkIsUnsupportedBlock` or similar instead, saves renaming it if we change the versions we support

Reply:
Done in [116e6c4](https://github.com/blockprotocol/blockprotocol/pull/417/commits/116e6c4a04c44e58d9dffcd9663b6cc0e5c5857b)